dags/adm_export.py

```diff
@@ -15,6 +15,9 @@
 This is a complementary approach to the near real-time sharing that is implemented
 in gcp-ingestion.
 
+Relies on the [`bq2stfp` container defined in `docker-etl`](https://github.com/mozilla/docker-etl/tree/main/jobs/bq2sftp)
+and credentials stored in the `adm_sftp` connection.
+
```

Comment:
This will appear in the "DAG Docs" banner of https://workflow.telemetry.mozilla.org/tree?dag_id=adm_export